### ISSUE TITLE
About Dataset Structure

### ISSUE THREAD

**Issue by idrisdemir** [guanyuezhen/MS-Former#2]
Hello,

I’ve been trying to train your model using the LEVIR-CD dataset, but I had a couple of small questions about the data setup. In train.py, the args.test_data_root argument seems to point to the validation set, not the actual test set — is that correct? Also, in test.py, you refer to a LEVIR-1024 directory. Does this contain the test data? And are there any differences in size or format compared to the training/validation images?

Thanks for your help.

**Comment by guanyuezhen** [comment]
> Hello,
> 
> I’ve been trying to train your model using the LEVIR-CD dataset, but I had a couple of small questions about the data setup. In train.py, the args.test_data_root argument seems to point to the validation set, not the actual test set — is that correct? Also, in test.py, you refer to a LEVIR-1024 directory. Does this contain the test data? And are there any differences in size or format compared to the training/validation images?
> 
> Thanks for your help.

1. For args.test_data_root,you can see that:
train_data = myDataLoader.Dataset(
    "train", file_root=args.train_data_root,
    transform=trainDataset_main, label_patch_size=args.patch_size
)
trainLoader = torch.utils.data.DataLoader(
    train_data,
    batch_size=args.batch_size, shuffle=True,
    num_workers=args.num_workers, pin_memory=False, drop_last=True
)

test_data = myDataLoader.Dataset(
    "test", file_root=args.test_data_root, transform=valDataset
)
testLoader = torch.utils.data.DataLoader(
    test_data, shuffle=False,
    batch_size=args.batch_size // 4, num_workers=args.num_workers, pin_memory=False
)
The "train" and "test" subsets represent the actual training and testing datasets.

2. We refer to a LEVIR-1024 directory because we aim to infer change results on the original 1024x1024 images, rather than on the cropped 256x256 images. Surely, the LEVIR-1024 directory contains the test data.
